### PR TITLE
fix: [SRM-12552]: Add brackets surrounding JSON path values

### PR DESCRIPTION
**Pull request by praneshg239**
#### Screenshots

Screenshot 2022-11-11 at 7 29 45 PM

Screenshot 2022-11-11 at 7 30 29 PM

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/85-cv/components/InputWithDynamicModalForJson/InputWithDynamicModalForJson.tsx [hunk 1]
```diff
@@ -80,9 +81,13 @@ export function InputWithDynamicModalForJson(props: InputWithDynamicModalForJson
           </Text>
           <JsonSelector
             json={sampleRecord as Record<string, any>}
-            onPathSelect={(pathSelected: string) => {
+            isSendSelectedRawJsonData
```
Comment (deepesh-ui) [inline src/modules/85-cv/components/InputWithDynamicModalForJson/InputWithDynamicModalForJson.tsx:84]: should this prop be always true ?

**Review by deepesh-ui — COMMENTED** [review]
Hope we have tested that this change is working fine with respective health sources.
Please check coverage

File: src/modules/85-cv/components/InputWithDynamicModalForJson/InputWithDynamicModalForJson.tsx (hunk 1 above)
Comment (praneshg239) [reply src/modules/85-cv/components/InputWithDynamicModalForJson/InputWithDynamicModalForJson.tsx]: Updated

**Review by deepesh-ui — APPROVED** [review]

**Review by harshilbits — APPROVED** [review]

**Comment by praneshg239** [comment]
Coverage is failing for old files, tests are covered for the changes in this PR.

**Pull request merged**